apps/frontend/src/libs/modules/store/store.module.ts

```diff
@@ -33,11 +35,14 @@ class Store {
 		this.instance = configureStore({
 			devTools: config.ENV.APP.ENVIRONMENT !== AppEnvironment.PRODUCTION,
 			middleware: (getDefaultMiddleware) => {
-				return getDefaultMiddleware({
-					thunk: {
-						extraArgument: this.extraArguments,
-					},
-				});
+				return [
+					...getDefaultMiddleware({
```

Comment:
Isn't `prepend` a preferred method for adding middleware?

Reply:
You're right